tools/ecfr-parser/ecfr/sections.go

```diff
@@ -0,0 +1,78 @@
+package ecfr
+
+type Part struct {
+	Name     string    `json:"name"`
+	Title    string    `json:"title"`
+	Sections []Section `json:"sections"`
+	Subparts []Subpart `json:"subparts"`
+}
+
+type Section struct {
+	Title   string `json:"title"`
+	Part    string `json:"part"`
+	Section string `json:"section"`
+}
+
+type Subpart struct {
+	Title    string    `json:"title"`
+	Part     string    `json:"part"`
+	Subpart  string    `json:"subpart"`
+	Sections []Section `json:"sections"`
+}
+
+func ExtractStructure(s Structure) (Part, error) {
+	title := s.Identifier[0]
+	structurePart := s.Children[0].Children[0].Children
```

Comment:
This doesn't feel great.   Why is it the third child?  Do we know the rest don't have multiple children?

Reply:
This is due to the structure pulled from eCFR. It is indeed a static thing because the top level is title, then chapter, then subchapter. See https://regulations-pilot.cms.gov/v2/2021-01-01/title/42

Reply:
ecfr.gov structure will always have the "part" be the third child. i.e https://www.ecfr.gov/api/versioner/v1/structure/2021-09-20/title-42.json?part=456
I could add in a check to make sure the 3rd child's type == "part"